task/buildah/0.3/buildah.yaml

```diff
@@ -93,7 +95,11 @@ spec:
 
   - name: digest-to-results
```

Comment:
Maybe not important, but you could revise the step name, e.g. "digest-and-url-to-results" or just "results" or "write-results"